README.md

```diff
@@ -21,16 +21,6 @@ We create and steward initiatives that press on the service mesh-centric envelop
 
 # Projects
 
-<p style="clear:both;">
-<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src="img/readme/meshery-operator.svg" style="margin:10px;" width="110px" align="left" /></a>
-# Meshery Operator
-Meshery Operator is the multi-service mesh operator and implementation of MeshSync.
-<br />
-<br />
-<br />
-</p>
-
-
 <p style="clear:both;">
```

Comment:
You can follow this way of adding the project description for Meshery Operator as well, to maintain consistency.